### DISCUSSION TITLE
Bun v0.1.1

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#206] [Announcements]
## What's New
* Web Streams - including `ReadableStream`, `WritableStream`, `TransformStream` and more. This was a massive project. There are still some reliability things to fix, but I'm very happy with the performance, and I think you will be too.
* [`WebSocket`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket) is now a global, powered by a custom WebSocket client
* Dynamic `require()`
* 50% faster TextEncoder
* ~30% faster React rendering in production due to a JSX transpiler optimization
* Streaming React SSR is now supported
* Dozens of small bug fixes to the HTTP server
* Discord interactions template by @evanwashere in https://github.com/Jarred-Sumner/bun/pull/2* Group zsh completion options by type by @alexkuz in https://github.com/Jarred-Sumner/bun/pull/194

## New Contributors
* @mjavadhpour made their first contribution in https://github.com/Jarred-Sumner/bun/pull/179
* @eswr made their first contribution in https://github.com/Jarred-Sumner/bun/pull/201

**Full Changelog**: https://github.com/Jarred-Sumner/bun/compare/bun-v0.0.83...bun-v0.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Jarred-Sumner/bun/releases/tag/bun-v0.1.1'>Bun v0.1.1</a>.</em>

**Comment by vitalspace** [comment]
Nice!!!

**Comment by Sparkenstein** [comment]
Any plan for windows releases?

**Comment by rafalohaki** [comment]
Love it

**Comment by LoiLock** [comment]
This is honestly becoming the coolest project on Github so fast right now.
